polaris/polaris/utils.py

```diff
@@ -135,7 +133,9 @@ def create_stellar_deposit(
         transaction.save()
         raise ValueError(transaction.status_message)
 
-    # if we don't know if the destination account exists
+    # if the destination account doesn't exists
```

Comment:
Did you feel the old comment wasn't clear? I think the 2nd condition in your new statement is redundant since thats true for any boolean function parameter that defaults to `False`.

`destination_exists=False` means we don't know if it exists, or rather that we need to check. If you want to rename it to something that makes more sense I'd support that.

Reply:
I'm planing on removing this logic and comment due to this change discussed here.
https://github.com/stellar/django-polaris/pull/341#discussion_r508100616